site.js

```diff
@@ -42,20 +65,73 @@ function placePhoto(url) {
         loadEvt.currentTarget.width,
         loadEvt.currentTarget.height,
         $box.width(),
-        $box.height()
+        $box.height(),
+        true
       );
 
       $img.
         attr('width', dims.width).
         attr('height', dims.height).
         attr('src', url).
+        data('nativeWidth', loadEvt.currentTarget.width).
+        data('nativeHeight', loadEvt.currentTarget.height).
         fadeIn('slow')
       ;
+
     });
   });
 }
 
 /**
+ * Dimensions of page
+ *
+ * @return {width: px, height: px}
+ */
+function getDocDims() {
+  var $body = $('body');
+  return {width: $body.width, height: $body.height()};
+}
+
+/**
+ * Blow up the image
+ */
+function zoomImage() {
+  var $ref = $('.photo img');
+
+  if ($ref.attr('src')) {
+    var speed = 'slow';
+    var twoEm = $('header').height();
```

Comment:
There's no way to get the size of an 'em' in JavaScript or even CSS. You just have to see what the browser does.